docs/resources/ocean_automatic_rightsizing.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: ocean_right_sizing_rule"
+subcategory: "Ocean"
+description: |-
+  Provides a Spotinst Ocean Right Sizing resource.
+---
+
+# spotinst\_ocean\_right\_sizing\_rule
+
+Manages a Spotinst Ocean right sizing rule resource.
+
+## Example Usage
+
+```hcl
+resource "spotinst_ocean_right_sizing_rule" "example" {
+  ocean_id = "o-123456"
+  rule_name = "test-rule"
+  exclude_preliminary_recommendations= true
+  restart_replicas="ALL_MANIFEST"
+
+  recommendation_application_hpa{
+    allow_hpa_recommendation= true
```

Comment:
please modify the variable name as discussed

Reply:
updated to allow_hpa_recommendations